invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -104,9 +98,9 @@ def get_affiliation(self, obj):
                     id_value = identifier["identifier"]
                     aff["affiliationIdentifier"] = id_value
                     aff["affiliationIdentifierScheme"] = scheme.upper()
-                    uri = self.URIS.get(scheme)
+                    uri = to_url(id_value, scheme)
                     if uri:
-                        aff["affiliationIdentifier"] = uri + id_value
+                        aff["affiliationIdentifier"] = uri
```

Comment:
This logic is not correct as it will always use the landing page URL instead of the normalised identifier.

- affiliationIdentifier: should be the normalised identifier value
- affilationIdentifierScheme: use the correct scheme acronym e.g. ``arXiv``

Reply:
I see the nameIdentifierScheme/affiliationIdentifierScheme is fixed in #756

Reply:
> affiliationIdentifier: should be the normalised identifier value

[Here](https://github.com/inveniosoftware/idutils/blob/master/idutils/__init__.py#L919) you can see the identifier value is normalized. Or do you mean that is hould **not have a URI** in it? This would result e.g. 

```xml
<affiliation affiliationIdentifier="01ggx4157" affiliationIdentifierScheme="ROR">CERN</affiliation>",
```

The rest of the comment I understand is about labelling/types that got fixed in a subsequent PR.

Reply:
Discussed IRL:
- Identifiers should be assumed to be normalized (Marshmallow-Utils: IdentifierScheme does this already)
- No URI should be calculated
- IDUtils normalization functions need to be reviewed (e.g. ROR should contain the URL)